hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/contract/TestRootedOzoneContract.java

```diff
@@ -1,55 +1,31 @@
-/**
+/*
  * Licensed to the Apache Software Foundation (ASF) under one
  * or more contributor license agreements.  See the NOTICE file
  * distributed with this work for additional information
  * regarding copyright ownership.  The ASF licenses this file
  * to you under the Apache License, Version 2.0 (the
  * "License"); you may not use this file except in compliance
  * with the License.  You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
+ * <p>
```

Comment:
Noise

Reply:
I didn't change license headers this way.

git thinks `TestRootedOzoneContractGetFileStatus.java` was renamed to `TestRootedOzoneContract.java` and changed like this.  But that's not the case.  `TestRootedOzoneContractGetFileStatus.java` was deleted, `TestRootedOzoneContract.java` is a completely new file.